README.md

```diff
@@ -20,14 +20,24 @@ For More Info on what a FOSS Contributor Fund is, and how to start your own, [Cl
 # Getting Started
 ### Prerequisites
 
-#### First, run npm install.
-
-#### Next, make a CSV with the Github Ids you're interested in checking, and, if desired, an "alternate id" to go with each
-The "alternate id" can be LDAPs, emails, or any other unique identifier you use for your people.
-You can also choose to not use this.
-##### Why CSV?
-We found that an easy way to get GitHub ids from Indeed employees was through a Google form that automatically recorded their Indeed email. From that form, we got a Google Sheet with employee info that is easily exported as a CSV. (Go to File, Download As, and then choose CSV.)  
-(Even if you're not using google forms to gather GitHub ids, you can still enter your data into a google sheet and get a CSV from that, if you're unfamiliar with the CSV format)
+#### First, clone the repo to your computer, then navigate to the starfish folder and run `npm install`.
+
+#### Next, make a .csv file with the Github Ids you're interested in checking, and, if desired, an "alternate id" to go with each
+> #### Q&A Time!
+> ##### What do you mean "alternate id"?
+> I mean, if you normally identify your employees by some unique identifier- like an LDAP, their email, an employee id number, or even just their names- you can add that to the CSV so that your output is more useful to you. (For example, we find a list of the emails of every employee who's eligible to vote in the FOSS Fund that month to be really useful.)  
+If you decide not to use an 'alternate id', your list of eligible employees will just be a list of their GitHub ids.
+> ##### Why CSV?
+> We found that an easy way to get GitHub ids from Indeed employees was through a Google form that automatically recorded their Indeed email. From that form, we got a Google Sheet with employee info that is easily exported as a CSV file. (Go to File, Download As, and then choose CSV.)
+> ##### How do I make a CSV?
+> A CSV is just a file of comma-separated values, with a carriage return between each line. It looks like this:  
+My GitHub ID is:,Email Address  
+danisyellis,dgellis@indeed.com  
+anexample,example@indeed.com  
```

Comment:
@example.com would probably be better here than example@indeed.com, since that's what it was designed for. 

Relatedly: Do we know that bm.gov and starfleet.com are not valid domains?

Reply:
changed to example@example.com
bm.gov doesn't seem to be. looks like starfleet.com was, at one time, an email domain. But, seriously, if you have bsisko@starfleet.com as your email address, you probably want people to email you like you're Captain Sisko, so I don't think I need to change that :-)